pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -493,3 +508,37 @@ func (p *Provisioner) populateDomainAndPort(sc *storagev1.StorageClass) error {
 
 	return nil
 }
+
+// Check for quota limit in OBC and set accordingly
+func (p Provisioner) SetObjectBucketClaimQuota(options *apibkt.BucketOptions) error {
```

Comment:
- How about using this method for all additional settings? Right now they just happen to be quota.
- The method name should be lower case since it's not needed outside this package.
```suggestion
func (p Provisioner) setAdditionalSettings(options *apibkt.BucketOptions) error {
```

Reply:
will address in next patch set